main/test/flix/Test.Exp.ArrayLit.flix

```diff
@@ -1,264 +1,351 @@
 namespace Test/Exp/ArrayLit {
 
     @test
-    def testArrayLit01(): Array[Unit] & Impure = []
+    def testArrayLit01(): ScopedArray[Unit, Impure] & Impure =
```

Comment:
I would like these to use regions such that the test becomes pure. You can just assign the array to an unused local and always return Unit.